app/src/test/java/com/hedvig/app/feature/marketing/ui/MarketingViewModelTest.kt

```diff
@@ -33,8 +30,8 @@ class MarketingViewModelTest {
   @get:Rule
   val mainCoroutineRule = MainCoroutineRule()
 
-  private fun sut(
-    marketManager: MarketManager = mockk(relaxed = true),
+  private fun createMarketingViewModel(
```

Comment:
Agree with steering away from `sut`. In hindsight I don't think I like it either, despite the fact that I wrote that in the first place.